{"uuid": "1f8a9c0d-8a9d-4c2e-bd10-eca668d19764", "children": ["7a165066-4d40-4514-b468-ecefa03f776d", "f1524189-84d3-42f0-b054-15ace41739ee", "8d376993-eff5-4982-9f79-3190562e9a5e", "9ac53070-af37-44d0-9439-439213a0ba43"], "befores": [{"name": "setup", "status": "passed", "start": 1693482804119, "stop": 1693482834428}], "afters": [{"name": "setup::0", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidSessionIdException: Message: invalid session id\nStacktrace:\n0   chromedriver                        0x000000010103265c chromedriver + 4318812\n1   chromedriver                        0x000000010102ad00 chromedriver + 4287744\n2   chromedriver                        0x0000000100c5c644 chromedriver + 296516\n3   chromedriver                        0x0000000100c8d940 chromedriver + 497984\n4   chromedriver                        0x0000000100cb763c chromedriver + 669244\n5   chromedriver                        0x0000000100cb6cb4 chromedriver + 666804\n6   chromedriver                        0x0000000100c2a0c0 chromedriver + 90304\n7   chromedriver                        0x0000000100ff2c40 chromedriver + 4058176\n8   chromedriver                        0x0000000100ff7160 chromedriver + 4075872\n9   chromedriver                        0x0000000100fbae68 chromedriver + 3829352\n10  chromedriver                        0x0000000100ff7c4c chromedriver + 4078668\n11  chromedriver                        0x0000000100fcff08 chromedriver + 3915528\n12  chromedriver                        0x0000000100c28ad8 chromedriver + 84696\n13  dyld                                0x0000000186533e50 start + 2544\n\n", "trace": "  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 911, in _teardown_yield_fixture\n    next(it)\n  File \"/Users/megha/PycharmProject/pywebautomationframework/tests/vwo_login/conftest.py\", line 14, in setup\n    driver.close()\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 456, in close\n    self.execute(Command.CLOSE)\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 345, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1693482846921, "stop": 1693482846924}], "start": 1693482804119, "stop": 1693482846924}